src/sonic-host-services/tests/hostcfgd/test_vectors.py

```diff
@@ -0,0 +1,287 @@
+from unittest.mock import call
+
+"""
+    hostcfgd test vector
+"""
+hostcfgdTestVector = [
```

Comment:
Constant names in Python code should be `ALL_CAPS_WITH_UNDERSCORES` (e.g., `HOSTCFGD_TEST_VECTOR`)